internal/terraform/node_resource_plan_instance.go

```diff
@@ -206,6 +206,22 @@ func (n *NodePlannableResourceInstance) managedResourceExecute(ctx EvalContext)
 			return diags
 		}
 
+		// TODO: it is currently important that we write resource changes to
```

Comment:
should we use FIXME here? since we're going track those specifically for taking care of known problems and technical dept issues

Reply:
Oops, good point! Fixed.